### PR TITLE
[DEV-1304] UX improvments

### PR DESCRIPTION
**Pull request by aleksandrmaksimoviclt**
Added:
- Live email validation
- Status messages
- Design options
- Icon accompany submit button

### CODE REVIEW

**Comment by aleksandrmaksimoviclt** [comment]
@simonwallstrom Translations would love to receive an update 👍

File: getanewsletter.php [hunk 1]
```diff
@@ -447,39 +502,61 @@ function news_js_ajax()
...
+                        'url': '<?php echo admin_url('admin-ajax.php'); ?>',
+                        'data': data,
+                        'cache': false,
+                        'beforeSend': function(message) {
+                            jQuery("#respond_message").html("&nbsp");
+                        },
+                        'success': function(response) {
+                            jQuery("#respond_message").html(success_message);
+                            jQuery("#id_email").val("");
+                            jQuery("#id_email").qcss({ 'border-color': 'unset' });
```
Comment (simonwallstrom) [inline getanewsletter.php]: "qcss" is it a spelling mistake or something I dont know about? :)

Comment (aleksandrmaksimoviclt) [inline getanewsletter.php]: Look a bit higher, i wrote a jQuery function, that combines queue and css

File: getanewsletter.php [hunk 2]
```diff
@@ -447,39 +502,61 @@ function news_js_ajax()
     <script type="text/javascript">
         //<![CDATA[
         jQuery(document).ready(function() {
-            jQuery('.news-loading').hide();
+
+            jQuery.fn.extend({
```
Comment (aleksandrmaksimoviclt) [inline getanewsletter.php]: @simonwallstrom Here

File: getanewsletter.php (hunk 1 above)
Comment (simonwallstrom) [inline getanewsletter.php]: aah, I see 👍

**Pull request closed without merging**